app/models/miq_alert.rb

```diff
@@ -545,17 +546,27 @@ def evaluate_in_automate(target, inputs = {})
   end
 
   def evaluate_internal(target, _inputs = {})
-    method = "evaluate_method_#{expression[:eval_method]}"
+    if target.class.name == 'MiddlewareServer'
```

Comment:
would that be expanded to other middleware classes in the future? I'm wondering if perhaps the `if` is a bit too specific.